src/main/java/seedu/address/ui/TeammateDashboard.java

```diff
@@ -57,11 +62,25 @@ public TeammateDashboard(Optional<Participation> teammate) {
         phone.setText("Teammate phone number: " + person.getPhone());
         email.setText("Teammate email: " + person.getEmail());
         address.setText("Teammate address: " + person.getAddress());
-        header1.setText("Projects participated: ");
-        person.getParticipations().keySet()
-                .forEach(participation -> projects.getChildren()
-                        .add(new Label(participation.toString())));
-        // TODO: Add person-project info
+        header1.setText("Tasks assigned: ");
+        for (Participation p : person.getParticipations().values()) {
+            System.out.println(p);
+            String projectName = p.getProject().getProjectName().toString();
+            Project proj = p.getProject();
+            Participation pp = proj.getParticipation(person.getGitUserNameString());
+            System.out.println(projectName);
+            for (Task task : pp.getTasks()) {
+                System.out.println(task);
+                listOfTasksName.add(projectName + ": " + task.getTaskName());
+            }
+        }
+        System.out.println("fasudhasodsa");
```

Comment:
Haha may delete this dummy debugging code.
There are several system output elsewhere. You may consider delete them all changing them to logging?